recipe/meta.yaml

```
@@ -154,15 +151,14 @@ outputs:
         - aiohttp >=3.7
         - aiohttp-cors
         - colorful
-        - gpustat >=1.0.0
+        - grpcio >=1.42.0
         - opencensus
         - prometheus_client >=0.7.1,<0.14.0
```

```suggestion
        - prometheus_client >=0.7.1
```
https://github.com/ray-project/ray/blob/ray-2.37.0/python/setup.py#L260

---

Nice catch, thanks ! Fixed in latest commit.